src/quo2/components/messages/system_message.cljs

```diff
@@ -70,7 +70,6 @@
    [rn/view
     {:align-self     :center
      :flex-direction :row
-     :margin-right   40 ;; dirty hack, flexbox won't work as expected
```

Comment:
@flexsurfer, I've taken out this line and encountered no problems. However, I could be overlooking something. Please review and inform me if its removal triggers any issues.

Reply:
i think that's fine, because you've changed the layout of elements and you have `:flex-shrink 0`